src/Fabfuel/Prophiler/View/toolbar.phtml

```diff
@@ -89,5 +97,27 @@ use Fabfuel\Prophiler\Toolbar\Formatter\BenchmarkFormatter;
         $('json').each(function (index, el) {
             el.innerHTML = JSON.stringify(JSON.parse(el.innerHTML), null, '    ');
         });
+
+        // Move toggle event to delegate handler
+        $('body')
+            .on('click', '#prophiler-toggle', function() {
+                $(this).find('.fa').toggleClass('fa-angle-up fa-angle-down');
```

Comment:
We can not assume, the user's project is using Font Awesome. 
The toggle button needs to work/be visible without it as well.